avm/res/insights/data-collection-rule/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -0,0 +1,166 @@
+targetScope = 'subscription'
+
+metadata name = 'WAF-aligned'
+metadata description = 'This instance deploys the module in alignment with the best-practices of the Azure Well-Architected Framework.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+// e.g., for a module 'network/private-endpoint' you could use 'dep-dev-network.privateendpoints-${serviceShort}-rg'
```

Comment:
Given that you have both an example for a `resourceGroupName` & `serviceShort` in the file - would you think we need additional examples?